spec/models/season/phase_switcher_spec.rb

```diff
@@ -54,7 +54,7 @@
     it 'fails silently when it receives a non-whitelisted phase' do
       phase = 'bad_intentions'
       forbidden_call = Season::PhaseSwitcher.destined(phase)
-      expect { forbidden_call }.not_to change { Season.current.updated_at }
+      expect { forbidden_call }.not_to change { Season.current.updated_at.to_s }
     end
```

Comment:
I got this error on Travis:
```when #doing_this_thing
     Failure/Error: expect { forbidden_call }.not_to change { Season.current.updated_at }
       expected result not to have changed, but did change from 2017-01-19 19:05:27.192892392 +0000 to 2017-01-19 19:05:27.192892000 +0000```
This has something to do with rounding. (Either that, or I am reaaaaally good at pointing the exact millinillisecond a few times )